modules/ravestate_telegramio/telegram_bot.py

```diff
@@ -22,9 +27,14 @@
 import ravestate_ontology
 
 
+MODULE_NAME: str = 'telegramio'
 TOKEN_CONFIG_KEY: str = "telegram-token"
+CHILD_CONN_CONFIG_KEY: str = "child_conn"
+CHILD_FILES_CONFIG_KEY: str = "child_config_files"
+ALL_IN_ONE_CONTEXT_CONFIG_KEY: str = 'all_in_one_context'
 
-active_chats: Set[int] = set()
+# contains only keys if all chats run in one process, else maps chat_id to Pipe
+active_chats: Dict[int, Union[mp.connection.Connection, None]] = dict()
```

Comment:
Please add a small comment explaining the difference between active chats and active users. (I know now, but it may not be obvious...)